testing/tests/ui/broken-config.stderr

```diff
@@ -0,0 +1,25 @@
+error: `$WORKSPACE/target/tests/trybuild/rinja_testing/no-such-config.toml` does not exist
+ --> tests/ui/broken-config.rs:4:38
+  |
+4 | #[template(source = "", ext = "txt", config = "no-such-config.toml")]
```

Comment:
I've been thinking quite a lot about this and I think we should use the span of the value and not of the key (so not `source` and `config` but the string literal). Because, especially in this case, it seems like we're saying there is something wrong with the attribute and not with its content.

Reply:
Yeah, I highlighted the key, not the value, because the key is shorter, and the value can be arbitrarily long. But I guess that's not a good reasoning. Changed.

Reply:
It's still they key here. :wink: